src/function_app.py

```
@@ -1,12 +1,20 @@
 import json
 import azure.functions as func
 import logging
-from readability.readability import Document
+from readability import Document
 import requests
 from bs4 import BeautifulSoup
 import random
 import time
 from urllib.parse import urlparse
```

Ruff (0.11.9)

`urllib.parse.urlparse` imported but unused

Remove unused import: `urllib.parse.urlparse`

(F401)